stripecardscan/src/main/java/com/stripe/android/stripecardscan/cardimageverification/CardImageVerificationActivity.kt

```diff
@@ -144,6 +147,12 @@ internal open class CardImageVerificationActivity :
      */
     private var imageConfigs: AcceptedImageConfigs = AcceptedImageConfigs()
 
+    /**
+     * The scan stats tracker for main loop duration
+     */
+    private var mainLoopStatsTracker: StatTracker? = null
+    private var currentScanPayloadInfo: PayloadInfo? = null
```

Comment:
maybe I'm misunderstanding. what's the purpose of having this as a class level variable?

Reply:
I won't claim to have a full picture of the path through the code, but where we get this is down in `cardReadyForVerification` and where we need to use it is over in `closeScanner`.  There were quite a few calls to closeScanner() that made passing it up the chain as a parameter got messy, especially since many of those calls are in the base class. That felt like too big a change. Let me know what you think.